tests/ui/test_ui_tools.py

```diff
@@ -1397,7 +1397,15 @@ def test_private_message_to_self(self, mocker):
         ('<hr/>', ['[RULER NOT RENDERED]']),
         ('<img>', ['[IMAGE NOT RENDERED]']),
         ('<img/>', ['[IMAGE NOT RENDERED]']),
-        ('<table>stuff</table>', ['[TABLE NOT RENDERED]']),
+        ('<table><thead><tr><th>Firstname</th><th>Lastname</th></tr></thead>'
+         '<tbody><tr><td>John</td><td>Doe</td></tr><tr><td>Mary</td><td>Moe'
+         '</td></tr></tbody></table>', [
+            '┌───────────┬──────────┐\n',
+            ('bold', '│ Firstname │ Lastname │\n'),
```

Comment:
I'm unsure about the best render, but it would seem better to me to have just the cell contents in `bold` or some other head-style, rather than the box parts too? This is particularly evident in the blue and light themes, where I first tested this.